com/haxepunk/Screen.hx

```diff
@@ -375,7 +375,7 @@ class Screen
 	 * @param	magnitude	Number of pixels to shake in any direction.
 	 * @param	duration	Duration of shake effect, in seconds.
 	 */
-	public function shake(magnitude:Int, duration:Float)
+	public function shake(magnitude:Int = 4, duration:Float = 0.5)
```

Comment:
I had a mess with using haxelib to point to my fork and pointing to the developer repo (automatically downloaded). I think I just got confused between Screen.hx and Scene.hx. I didnt see anything wrong with the change so I left it.